### DISCUSSION TITLE
VSCode Copilot Really Slow

### DISCUSSION THREAD

**Discussion by Shiran-Yuan** [community/community#19491] [Copilot Conversations] [answered]
It takes around 1-2 minutes for it to show one suggestion... Are there any fixes? Thanks!

**Comment by melvincwng** [comment, marked as answer]
@Shiran-Yuan, try the following:

1. In GitHub, go to 'Settings' --> go to 'Github Copilot' --> Ensure 'Suggestions matching public code' is allowed
2. Alternatively, when using Github Copilot in VSCode, press `Cltrl + Enter`. This command will show you multiple suggestions (if available).

Hope this resolves your issue.

If this answer has helped you, please kindly 'Mark as answer' so that others are able to view the solution, should they face the same problem in the future.

**Reply by Miladew** [reply]
> @Shiran-Yuan, coba yang berikut ini:
> 
> 1. Di GitHub, buka 'Pengaturan' --> buka 'Github Copilot' --> Pastikan 'Saran yang cocok dengan kode publik' diizinkan
> 2. Atau, saat menggunakan Github Copilot di VSCode, tekan `Cltrl + Enter`. Perintah ini akan menampilkan beberapa saran (jika tersedia).
> 
> Semoga ini menyelesaikan masalah Anda.
> 
> Jika jawaban ini membantu Anda, mohon 'Tandai sebagai jawaban' agar orang lain dapat melihat solusinya, jika mereka menghadapi masalah yang sama di masa mendatang.

**Comment by davecheney** [comment]
Can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.

**Reply by Miladew** [reply]
> Bisakah Anda menempelkan bagian View yang relevan -> Output; pilih _GitHub Copilot_ 

> Anda dapat menempelkan bagian View yang relevan -> Output; pilih _GitHub Copilot_ dari pulldown.

**Comment by davecheney** [comment]
@Shiran-Yuan I see you are in China. Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Reply by Miladew** [reply]
> @Shiran-YuanSaya melihat Anda berada di Cina. Apakah mungkin perusahaan Anda, penyedia layanan internet, atau pemerintah, mungkin mencegat lalu lintas SSL Anda?

**Comment by mmmmmmmgo** [comment]
<img width="1217" alt="image" src="https://user-images.githubusercontent.com/112670419/195975238-d698e5e0-db7a-40d4-9045-5d1910b74fb4.png">
Is this speed normal?

**Reply by davecheney** [reply]
@mmmmmmmgo it can be, depending on the language you are using and your location.

**Reply by Birdza87** [reply]
> <img width="1217" alt="image" src="https://user-images.githubusercontent.com/112670419/195975238-d698e5e0-db7a-40d4-9045-5d1910b74fb4.png">
> Is this speed normal?

**Reply by Birdza87** [reply]
Hi

**Reply by mmmmmmmgo** [reply]
> @mmmmmmmgo it can be, depending on the language you are using and your location.

<img width="932" alt="image" src="https://user-images.githubusercontent.com/112670419/196021275-6c644ace-1b7a-4211-a3f5-10cdd885bf4f.png">
